shell/platform/android/io/flutter/embedding/engine/dart/DartExecutor.java

```diff
@@ -287,10 +287,14 @@ public void notifyLowMemoryWarning() {
   }
 
   @Override
-  public void enableBufferingIncomingMessages() {}
+  public void enableBufferingIncomingMessages() {
```

Comment:
Move these methods to a place next to the other `BinaryMessenger` methods.

Also mark them as `@Deprecated`

Reply:
Done